jujugui/static/gui/src/app/components/profile/model-list/model-list.js

```diff
@@ -156,135 +167,122 @@ class ProfileModelList extends React.Component {
     @return {Object} The model list as JSX.
   */
   _generateModels() {
-    const icons = new Map([
-      ['read', 'show_16'],
-      ['write', 'edit_16'],
-      ['admin', 'user_16']
-    ]);
+    const icons = new Map([['read', 'show_16'], ['write', 'edit_16'], ['admin', 'user_16']]);
     const profileUsername = this.props.userInfo.profile;
     const models = this.state.models || [];
-    return models.reduce((modelList, model, index) => {
```

Comment:
I can't even tell what changed here...I'm going to stop reviewing here, can you revert these changes.